beginning.md

```diff
@@ -2,7 +2,7 @@
 
 ## Using Python interpreter
 
-* Type 'python3' command in terminal to open python interpreter
+* Type `python3` command in terminal to open a `Python Interpreter`.
```

Comment:
How about just keeping it as Python Interpreter? instead of highlighting it? We may just add single quotes around it.